### PR TITLE
Adding azure-controllers image build

### PR DESCRIPTION
**Pull request by kwoodson**
Adding the build for azure-controllers.

This will depend on https://github.com/openshift/openshift-azure/pull/693

### CODE REVIEW

**Comment by kwoodson** [comment]
rebased.

**Comment by kwoodson** [comment]
@kargakis, i tried to mirror angus' PR and your sync_image/image_sync variables.

**Comment by kargakis** [comment]
needs a rebase

**Comment by kwoodson** [comment]
@kargakis, rebased.  How does this look?

File: ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure-conformance.yaml [hunk 1]
```diff
@@ -23,6 +23,8 @@ parameters:
   required: true
 - name: TEST_COMMAND
   required: true
+- name: IMAGE_AZURE_CONTROLLERS
```
Comment (kargakis) [inline ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure-conformance.yaml]: This is already included in these parameters above.

File: projects/azure/image-mirror/image-mirror.yaml [hunk 2]
```diff
@@ -13,6 +13,7 @@ items:
       registry.svc.ci.openshift.org/azure/azure-plugins:etcdbackup quay.io/openshift-on-azure/etcdbackup:latest
       registry.svc.ci.openshift.org/azure/azure-plugins:azure-controllers quay.io/openshift-on-azure/azure-controllers:v3.11
       registry.svc.ci.openshift.org/azure/azure-misc:prometheus-sidecar quay.io/openshift-on-azure/prometheus-sidecar:latest
+      registry.svc.ci.openshift.org/azure/azure-plugins:azure-controllers quay.io/openshift-on-azure/azure-controllers:latest
```
Comment (kargakis) [inline projects/azure/image-mirror/image-mirror.yaml]: Seems to be included already two lines above.

**Pull request closed without merging**